src/client/containers/ModalCreateEvents/ModalCreateEvents.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Button } from '../../components/Button/Button';
+
+import './ModalCreateEvents.styles.css';
+import DropDown from '../../components/Dropdown/DropDown';
+import DateTime from '../../components/DateTime/DateTime';
+import Modal from '../../components/Modal/Modal';
+
+const companies = [
```

Comment:
You should push `companies` and `eventTypes` through props and use this hard-coded list in storybook.